Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4659,6 +4660,9 @@ def return_error(message, error='', outputs=None):
     if not isinstance(message, str):
         message = message.encode('utf8') if hasattr(message, 'encode') else str(message)
 
+    if is_command and demisto.command() == 'get-modified-remote-data' and not isinstance(error, NotImplementedError):
+        message += ' skip update'
```

Comment:
I would format it differently so it is clear that 'skip update' is not from the error message.
```suggestion
        message = 'skip update. error: ' + message
```